release-notes/aws/v10.1.2.md

```diff
@@ -0,0 +1,22 @@
+# :zap: Giant Swarm Release 10.1.2 for AWS is now active for you! :zap:
+
+This release fixes a problem with the cluster role for chart-operator, which is responsible for installing and updating Managed Apps. The changes affect only Managed Apps and upgrading to this release will not cause cluster nodes to be rolled. **To prevent encountering problems when installing or updating an app, please upgrade to this release.**
+
```

Comment:
```suggestion
Please note: We are still resolving the network traffic issue in 10.x and 11.x clusters reported yesterday.
```

Reply:
@pipo02mix @puja108 Added a disclaimer that this does not fix the issue Jessica reported yesterday. Looks ok?

@rossf7 I will share again in news-product w this change. But seems like nobody had any feedback on the previous one. Maybe we can merge this today?

Reply:
Here's the text Jessica used https://gigantic.slack.com/archives/C63HJK0CR/p1580899432066600

Reply:
@cokiengchiara See below I tweaked this slightly. If happy can you approve again?

I'll then merge this so we can ship it.

Reply:
> Added a disclaimer that this does not fix the issue Jessica reported yesterday. Looks ok?

I think it is good idea